docs/docs-nav.yml

```
@@ -2,17 +2,17 @@
 #       won't be highlighted.
   - title: Features
     children:
-      - title: "@ClearSystemProperty and @SetSystemProperty"
+      - title: "Clearing or Setting System Properties"
         url: /docs/system-properties/
-      - title: "@ClearEnvironmentVariable and @SetEnvironmentVariable"
+      - title: "Clearing or Setting Environment Variables"
         url: /docs/environment-variables/
-      - title: "@DefaultLocale and @DefaultTimeZone"
+      - title: "Default Locale and TimeZone"
         url: /docs/default-locale-timezone/
-      - title: "@DisableIfDisplayName"
+      - title: "Disable Based on DisplayName"
         url: /docs/disable-if-display-name/
       - title: "Range Sources"
         url: /docs/range-sources/
-      - title: "@RetryingTest"
+      - title: "Retrying Failing Tests"
         url: /docs/retrying-test/
       - title: "Report entries"
```

```suggestion
      - title: "Publishing Report Entries"
```

Then the ToC needs to be resorted.